openbb_platform/providers/fmp/openbb_fmp/models/index_historical.py

```diff
@@ -57,6 +57,12 @@ class FMPIndexHistoricalData(IndexHistoricalData):
         json_schema_extra={"x-unit_measurement": "percent", "x-frontend_multiply": 100},
     )
 
+    @field_validator("date", mode="before", check_fields=False)
+    @classmethod
+    def date_validate(cls, v):  # pylint: disable=E0213
+        """Return formatted datetime."""
+        return parse_date(v, "America/New_York")
```

Comment:
This will not work here, for the same reasons it won't in `equity.historical()`. They need to be TZ-unaware.